django/santropolFeast/member/models.py

```diff
@@ -176,8 +176,8 @@ class Meta:
         verbose_name_plural = _('clients')
 
     billing_address = models.ForeignKey(
-        'member.Address',
-        verbose_name=_('billing_Address')
+        'Address',
+        verbose_name=_('Billing address')
```

Comment:
Yeah it's a good point. But we need to keep the 'billing_address' ID to I18N.
It's not an english word, just an ID